### PR TITLE
Issue: 498 meta service long run fix

### PR DESCRIPTION
**Pull request by yamingk**
Problem Desc:
===========
We have a limitation on max chunk size (uint32_t) which is 4GB, which will be carved into portions in blk allocator with each portion maps to tens of thousands of nblks available for each portion. This specific test case specifiy max io size to to limitation that is larger than the portion total nblks and during the test, the io size is randomized and when it comes to larger than the limit, it can't find available allocation in any chunk so return failed.

Fix:
===
1. adjust the test case to not exceed limitation 
2. refactor the code a little to get max chunk size from Chunk class.

Test:
====
Specific failed test (previously failing consistently) can pass with this.

### CODE REVIEW

File: src/tests/test_scripts/log_meta_test.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def meta_nightly(options, addln_opts):
     subprocess.check_call(options.dirpath + "test_meta_blk_mgr " + cmd_opts + addln_opts, stderr=subprocess.STDOUT,
                           shell=True)
 
-    cmd_opts = "--min_write_size=10485760 --max_write_size=104857600 --bitmap=1"
+    cmd_opts = "--min_write_size=10485760 --max_write_size=80857600 --bitmap=1"
```
Comment (JacksonYao287) [inline src/tests/test_scripts/log_meta_test.py:84]: why `80857600`, is it upper limit of total nblks in the portion ?

Comment (yamingk) [reply src/tests/test_scripts/log_meta_test.py:84]: yes, it is a little smaller than the upper limit.

Comment (xiaoxichen) [reply src/tests/test_scripts/log_meta_test.py:84]: JFYI:  it translate to 2.4TB if blk size = 4K when storing bit map:) 

But maybe we are looking for 10x of this or even more.

**Review by xiaoxichen — APPROVED** [review]
lgtm

**Review by JacksonYao287 — APPROVED** [review]
LGTM

File: src/tests/test_scripts/log_meta_test.py (hunk 1 above)
Comment (yamingk) [reply src/tests/test_scripts/log_meta_test.py:84]: Thanks @xiaoxichen for bring it up. Yes, my thought is to create only one chunk for meta blk store, and change chunkz-size to uint64_t which should be big enough. 

Another thing is metablkstore has compression feature so the bitmap can be significantly compressed before written.

Comment (xiaoxichen) [reply src/tests/test_scripts/log_meta_test.py:84]: ok if it is per allocator then we are good?

Comment (yamingk) [reply src/tests/test_scripts/log_meta_test.py:84]: yes, we are good as in nublox bitmap.

**Pull request merged**